README.md

```diff
@@ -34,6 +34,44 @@ Use it as‑is or cherry‑pick only the folders you want.
 
 ## Quick start 🏁
 
+### Option 1: Automated Installation (Recommended) 🤖
+
+Install HVE ADO Scaffold into your existing repository using the PowerShell script:
+
+1. **Download and run the installer:**
+
+   ```powershell
```

Comment:
Should we include mac vs windows instructions?

Reply:
We may just want to add a `brew install powershell` if you're on Mac, and then direct them to call `pwsh` first before following these instructions?